pkgs/applications/networking/p2p/fragments/default.nix

```diff
@@ -1,76 +1,82 @@
 { lib
 , stdenv
 , fetchFromGitLab
-, meson
-, vala
-, ninja
-, pkg-config
-, wrapGAppsHook
-, desktop-file-utils
 , appstream-glib
-, python3
+, cmake
+, dbus
+, desktop-file-utils
+, git
 , glib
-, gtk3
-, libhandy
-, libtransmission
-, libb64
-, libutp
-, miniupnpc
-, dht
-, libnatpmp
-, libevent
-, curl
+, gtk4
+, libadwaita
+, meson
+, ninja
 , openssl
-, zlib
+, pkg-config
+, rustPlatform
+, sqlite
+, transmission
+, wrapGAppsHook4
 }:
 
 stdenv.mkDerivation rec {
   pname = "fragments";
-  version = "1.5";
+  version = "2.0.1";
 
   src = fetchFromGitLab {
     domain = "gitlab.gnome.org";
     owner = "World";
     repo = "Fragments";
     rev = version;
-    sha256 = "0x1kafhlgyi65l4w67c24r8mpvasg3q3c4wlgnjc9sxvp6ki7xbn";
+    sha256 = "sha256-3/v+MK7Zzn2tnUOe5cW35I5nJ+EsMUaqzTr25EloJ/k=";
   };
 
   patches = [
-    # Fix dependency resolution
-    ./dependency-resolution.patch
+    ./remove_meson_postinstall.patch
```

Comment:
https://gitlab.gnome.org/World/Fragments/-/merge_requests/129

Reply:
Is the `postinstall` script necessary, since nix does this as soon as you add it to the `systemPackages`, also this removes the `python3` dependency? Thank you for the PR.

Reply:
It would do `glib-compile-schemas` for us.

Reply:
What should I do? Keep it as is or use the script and use your PR as a patch?

Reply:
The patch is part of 2.0.2.

Reply:
I've updated to `2.0.2` and removed the `postinstall` patch.